rpa-manager/modules/ROOT/pages/processautomation-manage.adoc

```diff
@@ -1,8 +1,16 @@
 = Managing Automations
 
+An RPA project consists of a business process to be automated and a team that is responsible for the automation.
+
 When a project manager clicks the icon *Start automation* for a process in the *My Backlog* view of the *My RPA* module, RPA Manager creates a project for the automation of the process.
 
-To automate a process already evaluated with a third-party tool, a project manager can create an automation project inside a category directly in the *Process Automation* module.
+To automate a process already evaluated with a third-party tool, a project manager can alternativelely create an automation project directly in the following RPA Manager views:
+
+* RPA Home
```

Comment:
Are these UI elements? If so, they should be bold.
(Items in bold aren't translated.)